pkg/detectors/jiratoken/jiratoken_test.go

```diff
@@ -138,7 +138,7 @@ func TestJiraToken_FromChunk(t *testing.T) {
 					t.Fatalf("wantVerificationError = %v, verification error = %v", tt.wantVerificationErr, got[i].VerificationError)
 				}
 			}
-			ignoreOpts := cmpopts.IgnoreFields(detectors.Result{}, "Raw", "RawV2", "VerificationError")
+			ignoreOpts := cmpopts.IgnoreFields(detectors.Result{}, "Raw", "RawV2", "ExtraData", "VerificationError")
```

Comment:
It looks like the extra data that the detector is adding isn't dependent on the secret, so you can just add it to the test cases. Then you won't have to ignore this field in the comparison.